lib/ansible/modules/cloud/openstack/os_network.py

```diff
@@ -196,10 +207,12 @@ def main():
 
                 if project_id is not None:
                     net = cloud.create_network(name, shared, admin_state_up,
-                                               external, provider, project_id)
+                                               external, provider, project_id,
+                                               port_security_enabled)
```

Comment:
https://github.com/openstack/openstacksdk/blob/master/openstack/cloud/openstackcloud.py#L3382-L3386

`availability_zone_hints` comes after `project_id` so you need to fix the order, or use a keyword argument.

Reply:
done.